### PR TITLE
Feature view detailed club

### PR DESCRIPTION
**Pull request by 0us**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -34,4 +34,8 @@
     <string name="profile">Profile</string>
     <string name="sign_out">Sign out</string>
     <string name="join_club">Join club</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="title_activity_club_detailed">ClubDetailed</string>
```
Comment (0us) [inline app/src/main/res/values/strings.xml]: remove this

File: app/src/main/java/no/ntnu/klubbhuset/ui/main/MainActivity.java [hunk 2]
```diff
@@ -37,7 +31,17 @@ protected void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
-    public void onListFragmentInteraction(Club item) {
+    protected void onRestart() {
```
Comment (spiftire) [inline app/src/main/java/no/ntnu/klubbhuset/ui/main/MainActivity.java]: Do we really need to override this ?

File: app/src/main/java/no/ntnu/klubbhuset/ui/userviews/club/ClubDetailedFragment.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+public class ClubDetailedFragment extends Fragment {
+
+    public static ClubDetailedFragment newInstance(Club club) {
+        ClubDetailedFragment frag = new ClubDetailedFragment();
+        return frag;
+    }
+
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
```
Comment (spiftire) [inline app/src/main/java/no/ntnu/klubbhuset/ui/userviews/club/ClubDetailedFragment.java]: This is redundant

File: app/src/main/res/layout/content_club_detailed.xml [hunk 4]
```diff
@@ -2,9 +2,10 @@
...
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/club_detailed_fragment"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="no.ntnu.klubbhuset.ui.userviews.club.ClubDetailed">
+    tools:context=".ui.userviews.club.ClubDetailedFragment">
 
     <TextView
         android:id="@+id/textView9"
```
Comment (spiftire) [inline app/src/main/res/layout/content_club_detailed.xml]: ID naming is bad

File: app/src/main/java/no/ntnu/klubbhuset/data/model/Club.java [hunk 5]
```diff
@@ -3,12 +3,14 @@
...
 
+import java.io.Serializable;
+
 import lombok.AllArgsConstructor;
 import lombok.Data;
 
 @Data
 @AllArgsConstructor
-public class Club {
+public class Club implements Serializable {
```
Comment (markusranda) [inline app/src/main/java/no/ntnu/klubbhuset/data/model/Club.java]: Do we need Club implementing this interface?

**Pull request merged**